resources/benchmark-runner.js

```diff
@@ -1,15 +1,65 @@
-function BenchmarkTestStep(testName, testFunction) {
+class BenchmarkTestStep {
+constructor(testName, testFunction) {
     this.name = testName;
     this.run = testFunction;
 }
+}
+
+class BenchmarkState {
```

Comment:
Can we not change the order in which BenchmarkRunner and BenchmarkState are declared / defined?